wdtk-client/src/main/java/org/wikidata/wdtk/client/Client.java

```diff
@@ -204,6 +214,32 @@ private void openActions() {
 	}
 
 	/**
+	 * Creates a report file including the results which came from the
+	 * {@link DumpProcessingAction#getReport()} methods or printing it to
+	 * stdout.
```

Comment:
Please review this sentence (word order is mixed up). Also, it seems that you are never printing to stdout; you use the logger.